src/blog/v8-release-83.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'V8 release v8.3'
+author: 'Victor Gomes, safely working from home'
+avatars:
+ - 'victor-gomes'
+data: 2020-04-30
+tags:
+ - release
+description: 'V8 v8.3 features faster ArrayBuffers, bigger Wasm memories and deprecated APIs.'
+---
+
+Every six weeks, we create a new branch of V8 as part of our [release process](https://v8.dev/docs/release-process). Each version is branched from V8's Git master immediately before a Chrome Beta milestone. Today we're pleased to announce our newest branch, [V8 version 8.3](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/8.3), which is in beta until its release in coordination with Chrome 83 Stable in several weeks. V8 v8.3 is filled with all sorts of developer-facing goodies. This post provides a preview of some of the highlights in anticipation of the release.
+
+## Performance
+
+### Faster ArrayBuffer tracking in the garbage collector
+
+Backing stores of ArrayBuffers are allocated outside V8's heap using  `ArrayBuffer::Allocator` provided by the embedder. These backing stores need to be released when their ArrayBuffer object is reclaimed by the garbage collector. V8 8.3 has a new mechanism for tracking ArrayBuffers and their backing stores that allows the garbage collector to iterate and free the backing store concurrently to the application. More details are available in [this design document](https://docs.google.com/document/d/1-ZrLdlFX1nXT3z-FAgLbKal1gI8Auiaya_My-a0UJ28/edit#heading=h.gfz6mi5p212e). This reduced total GC pause time in ArrayBuffer heavy workloads by 50%.
+
+### Bigger Wasm memories
+
+In accordance with an update to the [WebAssembly specification](https://webassembly.github.io/spec/js-api/index.html#limits), V8 8.3 now allows modules to request memories up to 4GB in size, allowing more memory-heavy use cases to be brought to platforms powered by V8. Please keep in mind that this much memory might not always be available on a user’s system; we recommend creating memories at smaller sizes, growing them as needed, and gracefully handling failures to grow.
+
+## Fixes
+
+### Stores to objects with typed arrays on the prototype chain
+
+According to the JavaScript specification, when storing a value to the specified key we need to lookup the prototype chain to see if the key already exists on the prototype. More often than not these keys don't exist on the prototype chain, and so V8 installs fast lookup handlers to avoid these prototype chain walks when it is safe to do so.
+However, we recently identified a particular scenario where V8 incorrectly installed this fast lookup handler, leading to incorrect behaviour. When TypedArrays are on the prototype chain, all stores to keys which are OOB of the TypedArray should be ignored. For example, in the case below `v[2]` shouldn’t add a property to `v` and the subsequent reads should return undefined.
```

Comment:
Nit: if this a new paragraph, leave an empty line before for it to be rendered correctly.

Reply:
Done! 🙂